cmd/batch-handlers.go

```diff
@@ -557,6 +559,47 @@ func (r *BatchJobReplicateV1) Start(ctx context.Context, api ObjectLayer, job Ba
 			return false
 		}
 
+		if len(r.Flags.Filter.Tags) > 0 {
+			// Only parse object tags if tags filter is specified.
+			tagMap := map[string]string{}
```

Comment:
nit: `range` loop can handle `nil` map if object doesn't have any tags on it.
```suggestion
			var tagMap map[string]string
```

Reply:
it is kept for safety for lookups